### DISCUSSION TITLE
Gendern des Benutzerhandbuches

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#83] [Q&A]
Wir haben die Rückmeldung einer Nutzerin erhalten, dass das Benutzerhandbuch von WebGIS keine geschlechtergerechte Sprache verwende. Im Sinne der Geschlechtergleichstellung werden wir gebeten, darauf zu achten.
https://www.bundeskanzleramt.gv.at/agenda/frauen-und-gleichstellung/gleichbehandlung/sprachliche-gleichbehandlung/sprachliche-gleichbehandlung-frauen-maenner.html

**Comment by gis-eni** [comment]
Man kann den Pfad für die Online Hilfe theoretisch auf ein anderes Ziel umleiten und so auf eine eigene Hilfe ausweichen.

Die gesamte Online Dokumentation ist übrigens bereits OpenSource und liegt in diesem Repository: https://github.com/gis-eni/webgis-docs
Ihr seit gerne eingeladen, dort Änderungen vorzunehmen, die für alle anderen sinnvoll sind. Wie man hier vorgehen könnte, können wir auch beim nächsten Workshop/WebGIS Wednesday besprechen (fork => Pull Requests, ...)
